Solutions/Threat Intelligence/Hunting Queries/FileEntity_WireData.yaml

```diff
@@ -37,19 +36,27 @@ query: |
   | summarize WireData_TimeGenerated = arg_max(WireData_TimeGenerated, *) by IndicatorId, Process
   | project WireData_TimeGenerated, Description, ActivityGroupNames, IndicatorId, ThreatType, Url, ExpirationDateTime, ConfidenceScore,
   FileName, Computer, Direction, LocalIP, RemoteIP, LocalPortNumber, RemotePortNumber
-  | extend timestamp = WireData_TimeGenerated, HostCustomEntity = Computer, IPCustomEntity = RemoteIP, URLCustomEntity = Url
+  | extend timestamp = WireData_TimeGenerated, HostName = split(Computer, '.', 0)[0], DnsDomain = strcat_array(array_slice(split(Computer, '.'), 1, -1), '.')
+  | extend Host_0_HostName = HostName
+  | extend Host_0_DnsDomain = DnsDomain
+  | extend IP_0_Address = RemoteIP
+  | extend IP_1_Address = LocalIP
+  | extend URL_0_Url = Url
 entityMappings:
   - entityType: Host
     fieldMappings:
-      - identifier: FullName
-        columnName: HostCustomEntity
+      - identifier: HostName
+        columnName: HostName
+      - identifier: DnsDomain
+        columnName: DnsDomain
   - entityType: IP
     fieldMappings:
       - identifier: Address
-        columnName: IPCustomEntity
+        columnName: RemoteIP
```

Comment:
Since we have two IPs, there should be two separate entities for each one.

Reply:
Thanks @devikamehra for pointing this out. Done